themes/default/shop/config.json5

```diff
@@ -1,4 +1,5 @@
 {
+    // Needs updating
```

Comment:
what needs updating :D

Reply:
that whole f'in file lol. copypasta fortunately. good spot =)

Reply:
a9c902c